src/App.js

```diff
@@ -4,15 +4,17 @@ import "./App.css";
 import { Provider } from "react-redux";
 import store from "./store";
 import jwt_decode from "jwt-decode";
-import { setAuthToken } from "./utils/setAuthToken";
```

Comment:
if authorization is not required in header, remove the function too instead of just removing from here

Reply:
I did not understand.